Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.11
+
+RUN mkdir /incident_scraper
+
+COPY /incident_scraper /incident_scraper
+COPY pyproject.toml /incident_scraper
```

Comment:
Per my other comments below, changes to this step in the image will invalidate _all other steps below it_. This means that any time the code referenced in the `COPY` is changed, all steps below will need to be rerun. This can increase build time significantly for simple changes. I might recommend a setup like this:

```
FROM python:3.11

ENV PYTHONPATH=${PYTHONPATH}:${PWD}

RUN mkdir /incident_scraper
COPY pyproject.toml /incident_scraper

WORKDIR /incident_scraper

RUN pip3 install poetry && \
    poetry config virtualenvs.create false && \
    poetry install --no-dev

COPY /incident_scraper /incident_scraper
```

This step puts the most-frequently-changed step (the final `COPY`) at the _very_ end of the image definition, so all other layers can be cached for simple code changes!